app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -33,7 +33,12 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
     var data = response.data;
 
     $scope.afterGet = true;
+    $scope.cloudVolumeModel.emstype = data.emstype;
     $scope.cloudVolumeModel.name = data.name;
+    $scope.cloudVolumeModel.size = data.size;
+    // Currently, this is only relevant for AWS volumes so we are prefixing the
+    // model attribute with AWS.
+    $scope.cloudVolumeModel.aws_volume_type = data.volume_type;
```

Comment:
@gberginc We now have REST API support for the `cloud_volumes` collection...which means we can get rid of this `$http.get` call here and replace it with --
```url
/api/cloud_volumes/<id>?attributes=ext_management_system.type
```